site/content/docs/user/using-wsl2.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Using WSL2"
+menu:
+  main:
+    parent: "user"
+    identifier: "using-wsl2"
+    weight: 1
+---
+# Using WSL2
+
+KIND can run using Windows Subsystem for Linux 2 (WSL2) on Windows 10 Insider builds. All the tools needed to build or run KIND work in WSL2, but some extra steps are needed to switch to WSL2. This page covers these steps in brief but also links to the official documentation if you would like more details.
+
+## Getting Windows 10 Insider Preview
+
+Download latest ISO at https://www.microsoft.com/en-us/software-download/windowsinsiderpreviewadvanced . Choose "Windows 10 Insider Preview (FAST) - Build 18912". If there's a later build number, get that instead and then you can skip the work to upgrade to 18917.
+
+### VM Settings
```

Comment:
VM settings explain how to install Windows 10 Insider Preview in a VM? maybe `Installing on a virtual machine` is more explicit

Reply:
thanks! updated